### PR TITLE
Prevent double applying annotation when namespace annotation is used

### PR DESCRIPTION
**Pull request by ThetaDR**
Signed-off-by: Vyacheslav Tumanov <me@slavatumanov.me>

Related issue: https://github.com/networkservicemesh/deployments-k8s/issues/7676

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -139,7 +139,7 @@ linters:
     - errcheck
     - funlen
     - gochecknoinits
-    - goconst
+    # - goconst
```
Comment (denis-tingaikin) [inline .golangci.yml:142]: Can we disable it only for one file where we're using the default constants? See at example https://github.com/networkservicemesh/sdk/blob/main/.golangci.yml#L170

Comment (ThetaDR) [reply .golangci.yml]: Updated 👍

File: main.go [hunk 2]
```diff
@@ -86,7 +86,12 @@ func (s *admissionWebhookServer) Review(ctx context.Context, in *admissionv1.Adm
 	annotation := podMetaPtr.Annotations[s.config.Annotation]
 
 	// use namespace annotation only if resource doesn't have it's own
```
Comment (glazychev-art) [inline main.go:88]: Minor comment:
We can move this comment below

**Review by glazychev-art — APPROVED** [review]

File: main.go (hunk 2 above)
Comment (ThetaDR) [reply main.go]: I agree, forgot to move when new if statement was added. Fixed 👍

**Pull request merged**